README.md

```diff
@@ -1,21 +1,82 @@
 # a11y Viewer
 
-Simulate blindness, color blindness and disability to use mouse.
+**Emulate visual imparity, color blindness and inability to use a pointer.**
```

Comment:
Ok, then we should consistently update that. Also in GitHub repo and package.json description. @reinoute do you have time to make that change?